### PR TITLE
add new api changes to subscriptions rfc

### PR DESCRIPTION
**Pull request by a-alle**

### CODE REVIEW

File: docs/rfcs/rfc-007-subscriptions.md [hunk 1]
```diff
@@ -281,67 +283,559 @@ An example of the full payload:
...
             title
         }
         relationship {
             actors {
                 node {
                     name
                 }
+                edge {
+                    screenTime
+                }
```
Comment (darrellwarde) [inline docs/rfcs/rfc-007-subscriptions.md:492]: I would like to discuss this - whilst it is consistent with the filtering for Subscriptions, it is inconsistent with the rest of the API, which is a big thing we are trying to address at the moment. For instance, the filtering of Connection fields works the same as here, with `edge` and `node` at the same level - but then the data is returned in the format that I have suggested here. 

```suggestion
                screenTime
```

Comment (darrellwarde) [reply docs/rfcs/rfc-007-subscriptions.md:492]: That is correct!

File: docs/rfcs/rfc-007-subscriptions.md [hunk 2]
```diff
@@ -434,6 +960,122 @@ this0 { .name, actedIn: [ (this0)-[:ACTED_IN]->(this0_actedIn:Movie)   | this0_a
...
+        DETACH DELETE x
+        RETURN count(*) AS _
+}
+DETACH DELETE this
+WITH meta
+UNWIND meta AS m
+RETURN collect(DISTINCT m) AS meta
+```
+
+2. Uniform with disconnect impl version
```
Comment (a-alle) [inline docs/rfcs/rfc-007-subscriptions.md:1050]: @angrykoala would like your input on this: any preferences for either version 1 or 2?

File: docs/rfcs/rfc-007-subscriptions.md [hunk 3]
```diff
@@ -434,6 +960,156 @@ this0 { .name, actedIn: [ (this0)-[:ACTED_IN]->(this0_actedIn:Movie)   | this0_a
...
+
+-   match all relationships connected to the node, irrespective of the direction
+-   add to meta array
+
+```cypher
+WITH [] AS meta
+MATCH (this:`Movie`)
+WHERE this.title = $param0
+WITH this, meta + { event: "delete", id: id(this), properties: { old: this { .* }, new: null }, timestamp: timestamp(), typename: "Movie" } AS meta
+>>>
```
Comment (a-alle) [inline docs/rfcs/rfc-007-subscriptions.md:1011]: wrap in a CALL st

File: docs/rfcs/rfc-007-subscriptions.md [hunk 4]
```diff
@@ -434,6 +960,156 @@ this0 { .name, actedIn: [ (this0)-[:ACTED_IN]->(this0_actedIn:Movie)   | this0_a
...
+OPTIONAL MATCH (this)-[this_relationship:*]-(this_destination0)
+WITH this, meta, this_relationship, this_destination0
+WITH this,  this_relationship, this_destination0, meta + { event: "disconnect", id_from: id(this_destination0), id_to: id(this), id: id(this_relationship), properties: {from: this_destination0 {.*}, to: this {.*}, relationship: this_relationship {.*}} ,timestamp: timestamp(), relationshipName: type(this_relationship), fromTypename: labels(this_destination0), toTypename: labels(this) } AS meta
```
Comment (a-alle) [inline docs/rfcs/rfc-007-subscriptions.md:1014]: filter meta array by {id, event} in js

File: docs/rfcs/rfc-007-subscriptions.md [hunk 5]
```diff
@@ -309,53 +318,570 @@ subscription {
...
+-   ConnectOrCreate
+-   (out of scope) Currently, connection works as a idempotent operation with `MERGE`, the event should only trigger when a new connection is made.
+-   (Dis)Connections should be triggered regardless of the way the connection is created. (e.g. ActorUpdate or MovieUpdate as top level operation).
 -   (out of scope) Subscription to changes to the relationship properties are out of scope, only creation/deletion of relationships trigger events.
 
 #### Cypher Examples
 
 These are proposals on how to inject the given subscriptions into Cypher, the changes require for the source node, target node and relationship variables to be available on the metadata generation
 
+**!! UPDATE THESE !!**
```
Comment (angrykoala) [inline docs/rfcs/rfc-007-subscriptions.md:883]: :smile:

**Review by angrykoala — APPROVED** [review]

**Pull request merged**